src/main/java/io/stargate/sgv2/jsonapi/service/resolver/model/impl/InsertManyCommandResolver.java

```diff
@@ -38,7 +38,12 @@ public Operation resolveCommand(CommandContext ctx, InsertManyCommand command) {
 
     final DocumentProjector projection = ctx.indexingProjector();
     final List<WritableShreddedDocument> shreddedDocuments =
-        command.documents().stream().map(doc -> shredder.shred(doc, null, projection)).toList();
+        command.documents().stream()
+            .map(
+                doc ->
+                    shredder.shredWithMetrics(
+                        doc, null, projection, command.getClass().getSimpleName()))
+            .toList();
```

Comment:
I wonder... this will produce lots of individual metrics. I wonder if we should try to measure total time and size of all documents, instead of one by one?

Doing that would require more work here, to have `shredWithMetrics()` that takes `List` of documents, and then measures time over processing all, and counts total sum of size.

I think that would be more useful than having 20 different metrics point for a single command.

If we do that, it might also require reporting 3rd metric (number of Documents) -- which is actually useful metric on its own, I think. It'd be `1` for InsertOne, and then something else for `InsertMany`.